index.html

```diff
@@ -155,9 +177,7 @@ <h2 class="recentwork">
             <li><a href="#">Javascript</a></li>
             <li><a href="#">html</a></li>
           </ul>
-          <button class="resumebutton buttons" type="submit">
-            See Project
-          </button>
+          <button class="resumebutton buttons" type="submit">See Project</button>
         </article>
       </article>
```

Comment:
- [ ] Kindly add some space between the header and the badges to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206475837-a5bb7ed4-ada0-4a1d-bb86-8e0bb7813575.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206475917-17afe1b0-e235-49ea-b6a8-f4a8dda2cd65.png)